assets/out

```diff
@@ -94,7 +96,7 @@ if [ -n "$build" ]; then
     docker_pull "${repository}:${tag_name}"
   fi
 
-  docker build -t "${repository}:${tag_name}" "$build"
+  docker build -t "${repository}:${tag_name}" -f "$dockerfile" "$build"
```

Comment:
Will this work? Seems like it'd result in `docker build -t foo:bar -f Dockerfile ./foo`, when you really want `-f ./foo/Dockerfile` for the current behavior.

`docker build -h` makes it sound like it should be a path to the file, and isn't relative to the path:

```
  -f, --file=                     Name of the Dockerfile (Default is 'PATH/Dockerfile')
```